evalml/tests/automl_tests/test_automl.py

```diff
@@ -1819,7 +1819,10 @@ def test_pipelines_in_batch_return_nan(
             automl.search()
     assert len(automl.errors) > 0
     for pipeline_name, pipeline_error in automl.errors.items():
-        assert "Label Encoder" in pipeline_error["Parameters"]
+        assert (
+            "Label Encoder" in pipeline_error["Parameters"]
+            or "Mock Classifier" in pipeline_error["Parameters"]
+        )
```

Comment:
Same question here, why did this have to change?

Reply:
Previously, our mock didn't actually work. We now check to see that the mock classifier pipelines appear in the results.